src/pages/MovieList.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+const IMAGE_BASE_URL = "https://image.tmdb.org/t/p/";
+const POSTER_SIZE = "w500";
+
+function MovieList() {
+  const [movies, setMovies] = useState([]);
```

Comment:
Here you could also have another useState to mimic the loading behavior of the page